Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPIPermissions.mm

```diff
@@ -76,10 +115,10 @@ static void runScriptWithUserGesture(const String& script, WKWebView *background
         @"browser.test.assertTrue(await browser.permissions.remove({'permissions': ['cookies']}))",
 
         // Removing functional permissions should fail.
-        @"browser.test.assertRejects(browser.permissions.remove({'permissions': ['alarms']}))",
-        @"browser.test.assertRejects(browser.permissions.remove({'permissions': ['alarms', 'activeTab']}))",
-        @"browser.test.assertRejects(browser.permissions.remove({'permissions': ['cookies', 'activeTab']}))",
-        @"browser.test.assertRejects(browser.permissions.remove({'permissions': ['scripting']}))",
+        @"await browser.test.assertRejects(browser.permissions.remove({'permissions': ['alarms']}))",
```

Comment:
should these be moved to the Errors test?

Reply:
I kept them here for a more specific test. It is also tested in Errors.